addons/website/static/src/snippets/s_image_gallery/001.scss

```diff
@@ -0,0 +1,259 @@
+
+.o_gallery[data-vcss="001"] {
+    &.o_grid, &.o_masonry {
+        .img {
+            width: 100%;
+        }
+    }
+    &.o_grid {
+        &.o_spc-none div.row {
+            margin: 0;
+            > div {
+                padding: 0;
+            }
+        }
+        &.o_spc-small div.row > div {
+            padding: 0 5px;
```

Comment:
You cannot change the horizontal padding of column in standard bootstrap rows, you create misalignment with the other snippet/navbar containers.